### ISSUE TITLE
Difficult to navigate to/through setup nodes

### ISSUE THREAD

**Issue by yewang** [Kashomon/glift#117]
In viewing the Kogo's Joseki Dictionary (http://www.gliftgo.com/#Kogos), a significant part of the game tree is difficult to access. Directly after the root node, the ninth variation (marked with "I" at 10-3 on the right side) has only setup nodes (nodes that do not have a regular board move, but instead setup properties such as AB, AW, AE) as children. After clicking on I, it does not seem to be possible to progress onto any of those children using just the arrow keys (gui buttons or key presses). However, it seems that using "[" and "]" can invisibly select these setup node children, which can then be progressed onto via the right arrow key.

I guess this is more of a feature request/suggestion to make it easier to discover and navigate to/through setup nodes.

**Comment by Kashomon** [comment]
Yes, I agree.  Probably I need a to a make a tree viewer for situations like Kogos.  But I won't get to it for a little while due other priorities right now.

**Comment by yewang** [comment]
A tree viewer would be a nice feature, but there could also be other ways to expose variations and ease navigation.

For example, right now, one issue seems to be that the default next node just isn't set up properly in cases where the children are all setup nodes (I haven't checked if this is also an issue when the children are mixed between moves and setup). However, it seems that "[" and "]" still allows those child setup nodes to be selected and then moved to with the next node (or right arrow key press) button. So, a simple solution to make things a bit easier, would be to just figure out why the default next node is not being set up properly in this situation.

A relatively simple feature that may help in this situation would be the ability to toggle between siblings of the current node (perhaps with up and down arrow buttons/key presses).

Even without a full tree viewer, perhaps the UI could do something to show just a list of children and/or siblings (including empty and setup nodes) of the current node, with the ability to click on this list to navigate to the corresponding node.
